### ISSUE TITLE
Add sanity check for main class in the new project wizard

### ISSUE THREAD

**Issue by BlueRiverInteractive** [robovm/robovm-eclipse#42]
Only valid Java package/class names should be allowed. If illegal characters are added, the wizard should disable the next and finish buttons and display an error message.

Something like this should disable the buttons:
![illegal main class](http://i.imgur.com/cpjw2Dw.png)

**Comment by badlogic** [comment]
If we add an error label, we could probably also just add an explicit
package name text field.

**Comment by BlueRiverInteractive** [comment]
Yes, a package name text field would be good. If it's empty it will try to get the package from the main class. If both main class and package name have a package, the longest common string will be used as the package.
